scripts/deps-ppa/static_z3.sh

```diff
@@ -25,7 +25,7 @@ set -ev
 keyid=70D110489D66E2F6
 email=builds@ethereum.org
 packagename=libz3-static-dev
-version=4.8.9
+version=4.8.10
```

Comment:
How do we know if the build from this script succeeded?

Reply:
By checking that the tests in https://github.com/ethereum/solidity/pull/10845 do use z3 4.8.10 (the cmake log shows that)